src/components/progress/ProgressCard.jsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import { isFinite } from 'lodash-es';
+import { FormattedNumber } from 'react-intl';
+
+import LinearProgress from '@material-ui/core/LinearProgress';
+import Paper from '@material-ui/core/Paper';
+
+import Text from '../Text';
+
+export default function ProgressCard({ progress, style }) {
+  const isProgressFinite = isFinite(progress);
+  const progressPercentage = isProgressFinite ? progress * 100 : 0;
```

Comment:
Rounding to the nearest hundredth added in https://github.com/WildMeOrg/codex-frontend/pull/432/commits/6991952b4382fc589a090ad0e864a552e3fbf01c.

The `progressPercentage` was only the value passed to the `LinearProgress` component so that the visible progress bar would appear empty. If there is no `progress`, the text says "unknown progress"